src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
Although in your evidence there are contradictions and subjectivity, I'm not going to argue about this. Let's just put a point in this discussion! Let module include be placed at top of .cpp, but ORDERED includes should follow next.

P.S. Don't forget to add blank line between comment and #include.